dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -78,15 +83,29 @@ public void bindTo(StatementExecution cs, int ind) throws SQLException{
         this.cs=cs;
         this.index=ind;    
         if (direction != INPUT){
-            cs.registerOutParameter(ind, getSqlType());
+            cs.registerOutParameter(ind, getSqlType(), direction == RETURN_VALUE);
         }
     }
 
+    private Object transformValue(Object value) throws SQLException {
```

Comment:
> Yes, makes sense. I like transformToJdbcType or transformDbfitToJdbcType, aligned with the 'Type' part of the class and interface names.
> 

Hmm, here we got into another caveat - there is already distinction between `Java Object Types` and `SQL Types`:  `java.sql.Date` and `java.util.Date`. Basically the JDBC `setObject` converts between the the pure java and JDBC type (if needed). What we're doing is just a preparation yet we don't convert to the final JDBC type (as may be assumed by something named `transformToJdbcType`).

Also we already had short discussion about the `Type` part. (Could we have anther transformation which doesn't change the type).

> But do we use "JDBC" (as we have in some field/variable names) or "Jdbc"?

I think in general "Jdbc" would be easier to read. E.g. in case of multiple acronyms next to each other. (Some discussions on similar: http://stackoverflow.com/questions/1176950/acronyms-in-camel-back).